lib/dnsimple/client/clients.rb

```diff
@@ -57,7 +57,7 @@ def webhooks
     end
 
 
-    # @!class Struct
+    # Struct
```

Comment:
@jacegu I don't know about the original intent, but that isn't a valid YARDoc tag, which was causing a warning:

```shell
➜ yardoc
[warn]: Unknown directive @!class in file `lib/dnsimple/client/clients.rb` near line 61
# ...
```